cmc-domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ClaimData.java

```diff
@@ -7,14 +7,15 @@
 import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
 import org.hibernate.validator.constraints.NotBlank;
 import org.hibernate.validator.constraints.NotEmpty;
-import uk.gov.hmcts.cmc.claimstore.constraints.InterDependentFields;
-import uk.gov.hmcts.cmc.claimstore.models.amount.Amount;
-import uk.gov.hmcts.cmc.claimstore.models.legalrep.StatementOfTruth;
-import uk.gov.hmcts.cmc.claimstore.models.otherparty.TheirDetails;
-import uk.gov.hmcts.cmc.claimstore.models.particulars.HousingDisrepair;
-import uk.gov.hmcts.cmc.claimstore.models.particulars.PersonalInjury;
-import uk.gov.hmcts.cmc.claimstore.models.party.Party;
-import uk.gov.hmcts.cmc.claimstore.utils.MonetaryConversions;
+import uk.gov.hmcts.cmc.domain.models.amount.Amount;
+import uk.gov.hmcts.cmc.domain.models.constraints.InterDependentFields;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.models.particulars.HousingDisrepair;
+import uk.gov.hmcts.cmc.domain.models.particulars.PersonalInjury;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.utils.MonetaryConversions;
+import uk.gov.hmcts.cmc.domain.utils.ToStringStyle;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'uk.gov.hmcts.cmc.domain.utils.ToStringStyle'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AUselessImportCheck)